From: KevinDuan <duankaiwen@kylinos.com>
Date: Tue, 21 May 2024 02:23:35 +0000
Subject: Translated using Weblate (Uyghur)

Currently translated at 90.9% (121 of 133 strings)

Translation: openkylin-nile-new/kylin-music
Translate-URL: http://weblate.openkylin.top/projects/openkylin-nile-new/kylin-music/ug/
---
 translations/kylin-music_ug.ts | 242 ++++++++++++++++++++---------------------
 1 file changed, 121 insertions(+), 121 deletions(-)

diff --git a/translations/kylin-music_ug.ts b/translations/kylin-music_ug.ts
index bfb1d8d..69d3aef 100644
--- a/translations/kylin-music_ug.ts
+++ b/translations/kylin-music_ug.ts
@@ -7,7 +7,7 @@
         <location filename="../UI/base/allpupwindow.cpp" line="45"/>
         <location filename="../UI/base/allpupwindow.cpp" line="87"/>
         <source>Prompt information</source>
-        <translation type="unfinished"></translation>
+        <translation>ئەسكەرتمە ئۇچۇرلىرى</translation>
     </message>
     <message>
         <source>confirm</source>
@@ -20,32 +20,32 @@
     <message>
         <location filename="../UI/base/allpupwindow.cpp" line="45"/>
         <source>Could not contain characters:  / : * ? &quot; &amp; &lt; &gt; |</source>
-        <translation type="unfinished"></translation>
+        <translation>ئۆز ئىچىگە ئالمايدىغان تۆۋەندىكى ھەرىپلەر: / : * ? &quot; &amp; &lt; &gt; |</translation>
     </message>
     <message>
         <location filename="../UI/base/allpupwindow.cpp" line="53"/>
         <source>Reached upper character limit</source>
-        <translation type="unfinished"></translation>
+        <translation>ھەرىپ سانى يۇقىرى چەككە يەتتى</translation>
     </message>
     <message>
         <location filename="../UI/base/allpupwindow.cpp" line="62"/>
         <source>Music Player</source>
-        <translation type="unfinished"></translation>
+        <translation>مۇزىكا</translation>
     </message>
     <message>
         <location filename="../UI/base/allpupwindow.cpp" line="115"/>
         <source>Confirm</source>
-        <translation type="unfinished"></translation>
+        <translation>جەزىملەشتۈرۈش</translation>
     </message>
     <message>
         <location filename="../UI/base/allpupwindow.cpp" line="119"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>ئەمەلدىن قالدۇرۇش</translation>
     </message>
     <message>
         <location filename="../UI/base/allpupwindow.cpp" line="128"/>
         <source>Please input playlist name:</source>
-        <translation type="unfinished"></translation>
+        <translation>قويۇش تىزىملىكىنىڭ نامىنى كىرگۈزۈڭ:</translation>
     </message>
     <message>
         <source>Please enter the title of the playlist:</source>
@@ -58,23 +58,23 @@
         <location filename="../UIControl/base/musicfileinformation.cpp" line="229"/>
         <location filename="../UIControl/base/musicfileinformation.cpp" line="265"/>
         <source>Unknown singer</source>
-        <translation type="unfinished"></translation>
+        <translation>نامەلۇم ناخشىچى</translation>
     </message>
     <message>
         <location filename="../UIControl/base/musicfileinformation.cpp" line="230"/>
         <location filename="../UIControl/base/musicfileinformation.cpp" line="268"/>
         <source>Unknown album</source>
-        <translation type="unfinished"></translation>
+        <translation>نامەلۇم پىلاستىنكا</translation>
     </message>
     <message>
         <location filename="../UIControl/base/musicfileinformation.cpp" line="430"/>
         <source>Prompt information</source>
-        <translation type="unfinished"></translation>
+        <translation>ئەسكەرتمە ئۇچۇرلىرى</translation>
     </message>
     <message>
         <location filename="../UIControl/base/musicfileinformation.cpp" line="430"/>
         <source>Add failed, no valid music file found</source>
-        <translation></translation>
+        <translation>قوشۇش مەغلۇب بولۇش،ئىناۋەتلىك مۇزىكا ھۆججىتى تېپىلمىدى</translation>
     </message>
 </context>
 <context>
@@ -91,59 +91,59 @@
     <message>
         <location filename="../UI/base/allpupwindow.cpp" line="231"/>
         <source>Music Player</source>
-        <translation type="unfinished"></translation>
+        <translation>مۇزىكا</translation>
     </message>
     <message>
         <location filename="../UI/base/allpupwindow.cpp" line="297"/>
         <source>Confirm</source>
-        <translation type="unfinished"></translation>
+        <translation>جەزىملەشتۈرۈش</translation>
     </message>
     <message>
         <location filename="../UI/base/allpupwindow.cpp" line="300"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>ئەمەلدىن قالدۇرۇش</translation>
     </message>
     <message>
         <location filename="../UI/base/allpupwindow.cpp" line="247"/>
         <location filename="../UI/base/allpupwindow.cpp" line="323"/>
         <source>Song Name : </source>
-        <translation type="unfinished"></translation>
+        <translation>ناخشا ئىسمى : </translation>
     </message>
     <message>
         <location filename="../UI/base/allpupwindow.cpp" line="248"/>
         <location filename="../UI/base/allpupwindow.cpp" line="324"/>
         <source>Singer : </source>
-        <translation type="unfinished"></translation>
+        <translation>ناخشىچى : </translation>
     </message>
     <message>
         <location filename="../UI/base/allpupwindow.cpp" line="249"/>
         <location filename="../UI/base/allpupwindow.cpp" line="325"/>
         <source>Album : </source>
-        <translation type="unfinished"></translation>
+        <translation>ئالبۇم : </translation>
     </message>
     <message>
         <location filename="../UI/base/allpupwindow.cpp" line="250"/>
         <location filename="../UI/base/allpupwindow.cpp" line="326"/>
         <source>File Type : </source>
-        <translation type="unfinished"></translation>
+        <translation>ھۆججەت تۈرى : </translation>
     </message>
     <message>
         <location filename="../UI/base/allpupwindow.cpp" line="251"/>
         <location filename="../UI/base/allpupwindow.cpp" line="327"/>
         <source>File Size : </source>
-        <translation type="unfinished"></translation>
+        <translation>ھۆججەت چوڭلۇقى : </translation>
     </message>
     <message>
         <location filename="../UI/base/allpupwindow.cpp" line="252"/>
         <location filename="../UI/base/allpupwindow.cpp" line="328"/>
         <source>File Time : </source>
-        <translation type="unfinished"></translation>
+        <translation>ھۆججەت ۋاقتى : </translation>
     </message>
     <message>
         <location filename="../UI/base/allpupwindow.cpp" line="253"/>
         <location filename="../UI/base/allpupwindow.cpp" line="329"/>
         <source>File Path : </source>
-        <translation type="unfinished"></translation>
+        <translation>ھۆججەت يولى : </translation>
     </message>
 </context>
 <context>
@@ -151,22 +151,22 @@
     <message>
         <location filename="../UIControl/tableview/musiclistmodel.cpp" line="7"/>
         <source>Song</source>
-        <translation type="unfinished"></translation>
+        <translation>ناخشا</translation>
     </message>
     <message>
         <location filename="../UIControl/tableview/musiclistmodel.cpp" line="8"/>
         <source>Singer</source>
-        <translation type="unfinished"></translation>
+        <translation>ناخشىچى</translation>
     </message>
     <message>
         <location filename="../UIControl/tableview/musiclistmodel.cpp" line="9"/>
         <source>Album</source>
-        <translation type="unfinished"></translation>
+        <translation>ئالبۇم</translation>
     </message>
     <message>
         <location filename="../UIControl/tableview/musiclistmodel.cpp" line="10"/>
         <source>Time</source>
-        <translation type="unfinished"></translation>
+        <translation>ۋاقتىدا</translation>
     </message>
 </context>
 <context>
@@ -175,35 +175,35 @@
         <location filename="../UI/sidebar/mytoolbutton.cpp" line="142"/>
         <location filename="../UI/sidebar/mytoolbutton.cpp" line="155"/>
         <source>Play</source>
-        <translation type="unfinished"></translation>
+        <translation>قويۇش</translation>
     </message>
     <message>
         <location filename="../UI/sidebar/mytoolbutton.cpp" line="143"/>
         <location filename="../UI/sidebar/mytoolbutton.cpp" line="159"/>
         <source>Pause</source>
-        <translation type="unfinished"></translation>
+        <translation>توختاش</translation>
     </message>
     <message>
         <location filename="../UI/sidebar/mytoolbutton.cpp" line="144"/>
         <location filename="../UI/sidebar/mytoolbutton.cpp" line="163"/>
         <source>Rename</source>
-        <translation type="unfinished"></translation>
+        <translation>قايتا ئىسىم فامىلە قىلىش</translation>
     </message>
     <message>
         <location filename="../UI/sidebar/mytoolbutton.cpp" line="145"/>
         <location filename="../UI/sidebar/mytoolbutton.cpp" line="167"/>
         <source>Delete</source>
-        <translation type="unfinished"></translation>
+        <translation>ئۆچۈر</translation>
     </message>
     <message>
         <location filename="../UI/sidebar/mytoolbutton.cpp" line="173"/>
         <source>Song List</source>
-        <translation type="unfinished"></translation>
+        <translation>ناخشا تىزىملىكى</translation>
     </message>
     <message>
         <location filename="../UI/sidebar/mytoolbutton.cpp" line="173"/>
         <source>I Love</source>
-        <translation type="unfinished"></translation>
+        <translation>مەن ياخشى كۆرىمەن</translation>
     </message>
 </context>
 <context>
@@ -211,22 +211,22 @@
     <message>
         <location filename="../UI/player/playbackmodewidget.cpp" line="29"/>
         <source>Loop</source>
-        <translation type="unfinished"></translation>
+        <translation>دەۋرىيلىك ئايلانما قويۇش</translation>
     </message>
     <message>
         <location filename="../UI/player/playbackmodewidget.cpp" line="44"/>
         <source>Random</source>
-        <translation type="unfinished"></translation>
+        <translation>ئىختىيارىي قويۇش</translation>
     </message>
     <message>
         <location filename="../UI/player/playbackmodewidget.cpp" line="60"/>
         <source>Sequential</source>
-        <translation type="unfinished"></translation>
+        <translation>تەرتىپ بويىچە قويۇش</translation>
     </message>
     <message>
         <location filename="../UI/player/playbackmodewidget.cpp" line="75"/>
         <source>CurrentItemInLoop</source>
-        <translation type="unfinished"></translation>
+        <translation>تاق مۇزىكا ئايلانما قويۇش</translation>
     </message>
 </context>
 <context>
@@ -234,22 +234,22 @@
     <message>
         <location filename="../UI/player/playsongarea.cpp" line="32"/>
         <source>Previous</source>
-        <translation type="unfinished"></translation>
+        <translation>ئالدىنقى باسقۇچ</translation>
     </message>
     <message>
         <location filename="../UI/player/playsongarea.cpp" line="41"/>
         <source>Next</source>
-        <translation type="unfinished"></translation>
+        <translation>كېيىنكى باسقۇچ</translation>
     </message>
     <message>
         <location filename="../UI/player/playsongarea.cpp" line="45"/>
         <source>Volume</source>
-        <translation type="unfinished"></translation>
+        <translation>مۇشۇ جايدا رايونلارغا بۆلۈش</translation>
     </message>
     <message>
         <location filename="../UI/player/playsongarea.cpp" line="106"/>
         <source>Favourite</source>
-        <translation type="unfinished"></translation>
+        <translation>ياخشى كۆرۈش</translation>
     </message>
     <message>
         <location filename="../UI/player/playsongarea.cpp" line="133"/>
@@ -257,12 +257,12 @@
         <location filename="../UI/player/playsongarea.cpp" line="473"/>
         <location filename="../UI/player/playsongarea.cpp" line="519"/>
         <source>Loop</source>
-        <translation type="unfinished"></translation>
+        <translation>دەۋرىيلىك ئايلانما قويۇش</translation>
     </message>
     <message>
         <location filename="../UI/player/playsongarea.cpp" line="168"/>
         <source>Play List</source>
-        <translation type="unfinished"></translation>
+        <translation>قويۇش تارىخى</translation>
     </message>
     <message>
         <location filename="../UI/player/playsongarea.cpp" line="188"/>
@@ -272,18 +272,18 @@
         <location filename="../UI/player/playsongarea.cpp" line="875"/>
         <location filename="../UI/player/playsongarea.cpp" line="903"/>
         <source>Music Player</source>
-        <translation type="unfinished"></translation>
+        <translation>مۇزىكا</translation>
     </message>
     <message>
         <location filename="../UI/player/playsongarea.cpp" line="189"/>
         <source>00:00/00:00</source>
-        <translation type="unfinished"></translation>
+        <translation>00:00/00:00</translation>
     </message>
     <message>
         <location filename="../UI/player/playsongarea.cpp" line="416"/>
         <location filename="../UI/player/playsongarea.cpp" line="433"/>
         <source>I Love</source>
-        <translation type="unfinished"></translation>
+        <translation>مەن ياخشى كۆرىمەن</translation>
     </message>
     <message>
         <location filename="../UI/player/playsongarea.cpp" line="141"/>
@@ -291,7 +291,7 @@
         <location filename="../UI/player/playsongarea.cpp" line="482"/>
         <location filename="../UI/player/playsongarea.cpp" line="524"/>
         <source>Random</source>
-        <translation type="unfinished"></translation>
+        <translation>ئىختىيارىي قويۇش</translation>
     </message>
     <message>
         <source>Sequential</source>
@@ -303,7 +303,7 @@
         <location filename="../UI/player/playsongarea.cpp" line="499"/>
         <location filename="../UI/player/playsongarea.cpp" line="509"/>
         <source>CurrentItemInLoop</source>
-        <translation type="unfinished"></translation>
+        <translation>تاق مۇزىكا ئايلانما قويۇش</translation>
     </message>
 </context>
 <context>
@@ -311,7 +311,7 @@
     <message>
         <location filename="../UIControl/base/musicDataBase.cpp" line="64"/>
         <source>Database Error</source>
-        <translation type="unfinished"></translation>
+        <translation>ساندان خاتالىقى</translation>
     </message>
 </context>
 <context>
@@ -319,7 +319,7 @@
     <message>
         <location filename="../UI/player/searchedit.cpp" line="120"/>
         <source>Search Result</source>
-        <translation type="unfinished"></translation>
+        <translation>ئىزدەش نەتىجىسى</translation>
     </message>
 </context>
 <context>
@@ -327,17 +327,17 @@
     <message>
         <location filename="../UI/player/searchresult.cpp" line="33"/>
         <source>Music</source>
-        <translation type="unfinished"></translation>
+        <translation>ناخشا-مۇزىكا</translation>
     </message>
     <message>
         <location filename="../UI/player/searchresult.cpp" line="49"/>
         <source>Singer</source>
-        <translation type="unfinished"></translation>
+        <translation>ناخشىچى</translation>
     </message>
     <message>
         <location filename="../UI/player/searchresult.cpp" line="65"/>
         <source>Album</source>
-        <translation type="unfinished"></translation>
+        <translation>ئالبۇم</translation>
     </message>
 </context>
 <context>
@@ -345,50 +345,50 @@
     <message>
         <location filename="../UI/sidebar/sidebarwidget.cpp" line="55"/>
         <source>Music Player</source>
-        <translation type="unfinished"></translation>
+        <translation>مۇزىكا</translation>
     </message>
     <message>
         <location filename="../UI/sidebar/sidebarwidget.cpp" line="73"/>
         <source>Library</source>
-        <translation type="unfinished"></translation>
+        <translation>مۇزىكا ئامبىرى</translation>
     </message>
     <message>
         <location filename="../UI/sidebar/sidebarwidget.cpp" line="82"/>
         <source>Song List</source>
-        <translation type="unfinished"></translation>
+        <translation>ناخشا تىزىملىكى</translation>
     </message>
     <message>
         <location filename="../UI/sidebar/sidebarwidget.cpp" line="94"/>
         <source>My PlayList</source>
-        <translation type="unfinished"></translation>
+        <translation>قويۇش تىزىملىكىم</translation>
     </message>
     <message>
         <location filename="../UI/sidebar/sidebarwidget.cpp" line="139"/>
         <location filename="../UI/sidebar/sidebarwidget.cpp" line="345"/>
         <source>New Playlist</source>
-        <translation type="unfinished"></translation>
+        <translation>يېڭى قويۇلۇش تىزىملىكى</translation>
     </message>
     <message>
         <location filename="../UI/sidebar/sidebarwidget.cpp" line="143"/>
         <source>Rename</source>
-        <translation type="unfinished"></translation>
+        <translation>قايتا ئىسىم فامىلە قىلىش</translation>
     </message>
     <message>
         <location filename="../UI/sidebar/sidebarwidget.cpp" line="253"/>
         <source>I Love</source>
-        <translation type="unfinished"></translation>
+        <translation>مەن ياخشى كۆرىمەن</translation>
     </message>
     <message>
         <location filename="../UI/sidebar/sidebarwidget.cpp" line="304"/>
         <location filename="../UI/sidebar/sidebarwidget.cpp" line="392"/>
         <source>Prompt information</source>
-        <translation type="unfinished"></translation>
+        <translation>ئەسكەرتمە ئۇچۇرلىرى</translation>
     </message>
     <message>
         <location filename="../UI/sidebar/sidebarwidget.cpp" line="304"/>
         <location filename="../UI/sidebar/sidebarwidget.cpp" line="392"/>
         <source>Single song name already exists!!!</source>
-        <translation type="unfinished"></translation>
+        <translation>ناخشا تىزىملىكى ئاللىبۇرۇن مەۋجۇت!!</translation>
     </message>
 </context>
 <context>
@@ -404,27 +404,27 @@
     <message>
         <location filename="../UI/tableview/tablehistory.cpp" line="41"/>
         <source>History</source>
-        <translation type="unfinished"></translation>
+        <translation>قويۇش تارىخى</translation>
     </message>
     <message>
         <location filename="../UI/tableview/tablehistory.cpp" line="49"/>
         <source>Empty</source>
-        <translation type="unfinished"></translation>
+        <translation>قۇرۇق</translation>
     </message>
     <message>
         <location filename="../UI/tableview/tablehistory.cpp" line="67"/>
         <source>The playlist has no songs</source>
-        <translation type="unfinished"></translation>
+        <translation>مۇزىكا يوق</translation>
     </message>
     <message>
         <location filename="../UI/tableview/tablehistory.cpp" line="121"/>
         <source>Prompt information</source>
-        <translation type="unfinished"></translation>
+        <translation>ئەسكەرتمە ئۇچۇرلىرى</translation>
     </message>
     <message>
         <location filename="../UI/tableview/tablehistory.cpp" line="121"/>
         <source>Clear the playlist?</source>
-        <translation type="unfinished"></translation>
+        <translation>قويۇلۇش تىزىملىكىنى قۇرۇقدامسىز؟</translation>
     </message>
     <message>
         <location filename="../UI/tableview/tablehistory.cpp" line="122"/>
@@ -435,7 +435,7 @@
         <location filename="../UI/tableview/tablehistory.cpp" line="359"/>
         <location filename="../UI/tableview/tablehistory.cpp" line="390"/>
         <source>path does not exist</source>
-        <translation type="unfinished"></translation>
+        <translation>يول مەۋجۇت ئەمەس</translation>
     </message>
     <message>
         <source>Total </source>
@@ -444,22 +444,22 @@
     <message>
         <location filename="../UI/tableview/tablehistory.cpp" line="249"/>
         <source> songs</source>
-        <translation type="unfinished"></translation>
+        <translation> ئالدىدىكى ناخشا</translation>
     </message>
     <message>
         <location filename="../UI/tableview/tablehistory.cpp" line="278"/>
         <source>Play</source>
-        <translation type="unfinished"></translation>
+        <translation>قويۇش</translation>
     </message>
     <message>
         <location filename="../UI/tableview/tablehistory.cpp" line="279"/>
         <source>Delete</source>
-        <translation type="unfinished"></translation>
+        <translation>ئۆچۈر</translation>
     </message>
     <message>
         <location filename="../UI/tableview/tablehistory.cpp" line="280"/>
         <source>Play the next one</source>
-        <translation type="unfinished"></translation>
+        <translation>كېيىنكىسىنى قويۇش</translation>
     </message>
     <message>
         <source>The song doesn&apos;t exist</source>
@@ -494,22 +494,22 @@
         <location filename="../UI/tableview/tableone.cpp" line="1134"/>
         <location filename="../UI/tableview/tableone.cpp" line="1206"/>
         <source>Song List</source>
-        <translation type="unfinished"></translation>
+        <translation>ناخشا تىزىملىكى</translation>
     </message>
     <message>
         <location filename="../UI/tableview/tableone.cpp" line="191"/>
         <source>There are no songs!</source>
-        <translation type="unfinished"></translation>
+        <translation>مۇزىكا يوق</translation>
     </message>
     <message>
         <location filename="../UI/tableview/tableone.cpp" line="147"/>
         <source>Play All</source>
-        <translation type="unfinished"></translation>
+        <translation>ھەممىنى قويۇش</translation>
     </message>
     <message>
         <location filename="../UI/tableview/tableone.cpp" line="150"/>
         <source>Add Music</source>
-        <translation type="unfinished"></translation>
+        <translation>مۇزىكا قوشۇش</translation>
     </message>
     <message>
         <location filename="../UI/tableview/tableone.cpp" line="156"/>
@@ -538,7 +538,7 @@
         <location filename="../UI/tableview/tableone.cpp" line="1095"/>
         <location filename="../UI/tableview/tableone.cpp" line="1136"/>
         <source>I Love</source>
-        <translation type="unfinished"></translation>
+        <translation>مەن ياخشى كۆرىمەن</translation>
     </message>
     <message>
         <location filename="../UI/tableview/tableone.cpp" line="263"/>
@@ -548,27 +548,27 @@
         <location filename="../UI/tableview/tableone.cpp" line="1232"/>
         <location filename="../UI/tableview/tableone.cpp" line="1261"/>
         <source>Search Result</source>
-        <translation type="unfinished"></translation>
+        <translation>ئىزدەش نەتىجىسى</translation>
     </message>
     <message>
         <location filename="../UI/tableview/tableone.cpp" line="291"/>
         <source>Play</source>
-        <translation type="unfinished"></translation>
+        <translation>قويۇش</translation>
     </message>
     <message>
         <location filename="../UI/tableview/tableone.cpp" line="292"/>
         <source>Delete from list</source>
-        <translation type="unfinished"></translation>
+        <translation>تىزىمدىن ئۆچۈرۈش</translation>
     </message>
     <message>
         <location filename="../UI/tableview/tableone.cpp" line="293"/>
         <source>Remove from local</source>
-        <translation type="unfinished"></translation>
+        <translation>يەرلىكتىن چىقىرىۋېتىش</translation>
     </message>
     <message>
         <location filename="../UI/tableview/tableone.cpp" line="368"/>
         <source>Confirm that the selected song will be deleted from the song list?</source>
-        <translation type="unfinished"></translation>
+        <translation>تاللانغان ناخشىنى ناخشا تىزىملىكىدىن ئۆچۈرۈلىدىغانلىقىنى جەزىملەشتۈرەمسىز؟</translation>
     </message>
     <message>
         <location filename="../UI/tableview/tableone.cpp" line="369"/>
@@ -591,7 +591,7 @@
         <location filename="../UI/tableview/tableone.cpp" line="943"/>
         <location filename="../UI/tableview/tableone.cpp" line="961"/>
         <source>Yes</source>
-        <translation type="unfinished"></translation>
+        <translation>شۇنداق</translation>
     </message>
     <message>
         <location filename="../UI/tableview/tableone.cpp" line="371"/>
@@ -599,19 +599,19 @@
         <location filename="../UI/tableview/tableone.cpp" line="524"/>
         <location filename="../UI/tableview/tableone.cpp" line="625"/>
         <source>No</source>
-        <translation type="unfinished"></translation>
+        <translation>ياق</translation>
     </message>
     <message>
         <location filename="../UI/tableview/tableone.cpp" line="395"/>
         <location filename="../UI/tableview/tableone.cpp" line="521"/>
         <location filename="../UI/tableview/tableone.cpp" line="622"/>
         <source>After the song is deleted from the local, it cannot be resumed. Is it sure to delete?</source>
-        <translation type="unfinished"></translation>
+        <translation>ناخشا ئۆچۈرۈلگەندىن كېيىن ئەسلىگە كەلتۈرۈشكە بولمايدۇ. راستىنلا ئۆچۈرەمسىز؟</translation>
     </message>
     <message>
         <location filename="../UI/tableview/tableone.cpp" line="775"/>
         <source>Audio File</source>
-        <translation type="unfinished"></translation>
+        <translation>ئۈن-سىن ھۆججىتى</translation>
     </message>
     <message>
         <location filename="../UI/tableview/tableone.cpp" line="921"/>
@@ -621,12 +621,12 @@
     <message>
         <location filename="../UI/tableview/tableone.cpp" line="959"/>
         <source>Repeat add</source>
-        <translation type="unfinished"></translation>
+        <translation>قوشۇشنى تەكرارلاش</translation>
     </message>
     <message>
         <location filename="../UI/tableview/tableone.cpp" line="995"/>
         <source>path does not exist</source>
-        <translation type="unfinished"></translation>
+        <translation>يول مەۋجۇت ئەمەس</translation>
     </message>
     <message>
         <source>Are you sure you want to delete it locally?</source>
@@ -636,12 +636,12 @@
         <location filename="../UI/tableview/tableone.cpp" line="930"/>
         <location filename="../UI/tableview/tableone.cpp" line="952"/>
         <source>Success add %1 songs</source>
-        <translation type="unfinished"></translation>
+        <translation>1%ناخشىنى مۇۋاپپىقىيەتلىك قوشۇش</translation>
     </message>
     <message>
         <location filename="../UI/tableview/tableone.cpp" line="941"/>
         <source>Add failed</source>
-        <translation type="unfinished"></translation>
+        <translation>قوشۇش مەغلۇپ بولدى</translation>
     </message>
     <message>
         <source>Delete</source>
@@ -650,12 +650,12 @@
     <message>
         <location filename="../UI/tableview/tableone.cpp" line="294"/>
         <source>View song information</source>
-        <translation type="unfinished"></translation>
+        <translation>ناخشا ئۇچۇرلىرى</translation>
     </message>
     <message>
         <location filename="../UI/tableview/tableone.cpp" line="295"/>
         <source>Add to songlist</source>
-        <translation type="unfinished"></translation>
+        <translation>ناخشا تىزىملىكىگە قوش</translation>
     </message>
     <message>
         <location filename="../UI/tableview/tableone.cpp" line="368"/>
@@ -667,7 +667,7 @@
         <location filename="../UI/tableview/tableone.cpp" line="952"/>
         <location filename="../UI/tableview/tableone.cpp" line="959"/>
         <source>Prompt information</source>
-        <translation type="unfinished"></translation>
+        <translation>ئەسكەرتمە ئۇچۇرلىرى</translation>
     </message>
     <message>
         <source>Add failed!</source>
@@ -684,7 +684,7 @@
     <message>
         <location filename="../UI/tableview/tableone.cpp" line="1031"/>
         <source> songs</source>
-        <translation type="unfinished"></translation>
+        <translation> ئالدىدىكى ناخشا</translation>
     </message>
 </context>
 <context>
@@ -692,12 +692,12 @@
     <message>
         <location filename="../UI/titlebar/titlebar.cpp" line="57"/>
         <source>back</source>
-        <translation type="unfinished"></translation>
+        <translation>يىغىش</translation>
     </message>
     <message>
         <location filename="../UI/titlebar/titlebar.cpp" line="69"/>
         <source>forward</source>
-        <translation type="unfinished"></translation>
+        <translation>قوشۇش</translation>
     </message>
     <message>
         <source>Search for music, singers</source>
@@ -714,27 +714,27 @@
     <message>
         <location filename="../UI/titlebar/titlebar.cpp" line="92"/>
         <source>Not logged in</source>
-        <translation type="unfinished"></translation>
+        <translation>تىزىملىتىلمىغان</translation>
     </message>
     <message>
         <location filename="../UI/titlebar/titlebar.cpp" line="155"/>
         <source>mini model</source>
-        <translation type="unfinished"></translation>
+        <translation>ئاددىيلاشتۇرۇلغان شەكلى</translation>
     </message>
     <message>
         <location filename="../UI/titlebar/titlebar.cpp" line="164"/>
         <source>To minimize the</source>
-        <translation type="unfinished"></translation>
+        <translation>كىچىكلىتىش</translation>
     </message>
     <message>
         <location filename="../UI/titlebar/titlebar.cpp" line="175"/>
         <source>maximize</source>
-        <translation type="unfinished"></translation>
+        <translation>چوڭايتىش</translation>
     </message>
     <message>
         <location filename="../UI/titlebar/titlebar.cpp" line="186"/>
         <source>close</source>
-        <translation type="unfinished"></translation>
+        <translation>يېپىش</translation>
     </message>
 </context>
 <context>
@@ -742,7 +742,7 @@
     <message>
         <location filename="../UI/mainwidget.cpp" line="599"/>
         <source>Music Player</source>
-        <translation type="unfinished"></translation>
+        <translation>مۇزىكا</translation>
     </message>
     <message>
         <source>Song List</source>
@@ -752,18 +752,18 @@
         <location filename="../UI/mainwidget.cpp" line="848"/>
         <location filename="../UI/mainwidget.cpp" line="1047"/>
         <source>reduction</source>
-        <translation type="unfinished"></translation>
+        <translation>ئەسلىگە قايتۇرۇش</translation>
     </message>
     <message>
         <location filename="../UI/mainwidget.cpp" line="854"/>
         <location filename="../UI/mainwidget.cpp" line="1039"/>
         <source>maximize</source>
-        <translation type="unfinished"></translation>
+        <translation>چوڭايتىش</translation>
     </message>
     <message>
         <location filename="../UI/mainwidget.cpp" line="1173"/>
         <source>Prompt information</source>
-        <translation type="unfinished"></translation>
+        <translation>ئەسكەرتمە ئۇچۇرلىرى</translation>
     </message>
 </context>
 <context>
@@ -771,45 +771,45 @@
     <message>
         <location filename="../UI/titlebar/menumodule.cpp" line="32"/>
         <source>Menu</source>
-        <translation type="unfinished"></translation>
+        <translation>تىزىملىك</translation>
     </message>
     <message>
         <location filename="../UI/titlebar/menumodule.cpp" line="44"/>
         <source>Theme</source>
-        <translation type="unfinished"></translation>
+        <translation>ئۇسلۇب</translation>
     </message>
     <message>
         <location filename="../UI/titlebar/menumodule.cpp" line="46"/>
         <location filename="../UI/titlebar/menumodule.cpp" line="150"/>
         <source>Help</source>
-        <translation type="unfinished"></translation>
+        <translation>ياردەم</translation>
     </message>
     <message>
         <location filename="../UI/titlebar/menumodule.cpp" line="48"/>
         <location filename="../UI/titlebar/menumodule.cpp" line="148"/>
         <source>About</source>
-        <translation type="unfinished"></translation>
+        <translation>ھەققىدە</translation>
     </message>
     <message>
         <location filename="../UI/titlebar/menumodule.cpp" line="50"/>
         <location filename="../UI/titlebar/menumodule.cpp" line="146"/>
         <source>Exit</source>
-        <translation type="unfinished"></translation>
+        <translation>چىقىش ئېغىزى</translation>
     </message>
     <message>
         <location filename="../UI/titlebar/menumodule.cpp" line="57"/>
         <source>Auto</source>
-        <translation type="unfinished"></translation>
+        <translation>ئاپتۇماتىك</translation>
     </message>
     <message>
         <location filename="../UI/titlebar/menumodule.cpp" line="62"/>
         <source>Light</source>
-        <translation type="unfinished"></translation>
+        <translation>سۇس</translation>
     </message>
     <message>
         <location filename="../UI/titlebar/menumodule.cpp" line="67"/>
         <source>Dark</source>
-        <translation type="unfinished"></translation>
+        <translation>قېنىق</translation>
     </message>
     <message>
         <source>Quit</source>
@@ -819,12 +819,12 @@
         <location filename="../UI/titlebar/menumodule.cpp" line="254"/>
         <location filename="../UI/titlebar/menumodule.cpp" line="295"/>
         <source>Music Player</source>
-        <translation type="unfinished"></translation>
+        <translation>مۇزىكا</translation>
     </message>
     <message>
         <location filename="../UI/titlebar/menumodule.cpp" line="289"/>
         <source>Music player is a multimedia playback software.Cover Various music formats Playback tools for,fast and simple.</source>
-        <translation type="unfinished"></translation>
+        <translation>مۇزىكا قويغۇچ بولسا ھەر خىل شەكىلدىكى مۇزىكىلارنى كۆپ ۋاستىلىك قويۇش دېتالىدا ئىشلىتىلىدۇ. ئۇ ھەر خىل مۇزىكا شەكلىدىكى قويۇش قوراللىرىنى ئۆز ئىچىگە ئالدۇ، مەشغۇلاتى تېز ۋە ئاددىي بولىدۇ.</translation>
     </message>
     <message>
         <source>Music player is a multimedia playback software.Cover Various music formats Playback tools for,Fast and simple.</source>
@@ -837,13 +837,13 @@
     <message>
         <location filename="../UI/titlebar/menumodule.cpp" line="299"/>
         <source>Version: </source>
-        <translation type="unfinished"></translation>
+        <translation>نەشرى: </translation>
     </message>
     <message>
         <location filename="../UI/titlebar/menumodule.cpp" line="356"/>
         <location filename="../UI/titlebar/menumodule.cpp" line="364"/>
         <source>Service &amp; Support: </source>
-        <translation type="unfinished"></translation>
+        <translation>مۇلازىمەت ۋە قوللىغان كوللېكتىپ </translation>
     </message>
 </context>
 <context>
@@ -854,7 +854,7 @@
         <location filename="../UI/player/miniwidget.cpp" line="614"/>
         <location filename="../UI/player/miniwidget.cpp" line="719"/>
         <source>Music Player</source>
-        <translation type="unfinished"></translation>
+        <translation>مۇزىكا</translation>
     </message>
     <message>
         <location filename="../UI/player/miniwidget.cpp" line="60"/>
@@ -880,19 +880,19 @@
     <message>
         <location filename="../UI/player/miniwidget.cpp" line="269"/>
         <source>00:00/00:00</source>
-        <translation type="unfinished"></translation>
+        <translation>00:00/00:00</translation>
     </message>
     <message>
         <location filename="../UI/player/miniwidget.cpp" line="330"/>
         <location filename="../UI/player/miniwidget.cpp" line="697"/>
         <source>CurrentItemInLoop</source>
-        <translation type="unfinished"></translation>
+        <translation>تاق مۇزىكا ئايلانما قويۇش</translation>
     </message>
     <message>
         <location filename="../UI/player/miniwidget.cpp" line="509"/>
         <location filename="../UI/player/miniwidget.cpp" line="525"/>
         <source>I Love</source>
-        <translation type="unfinished"></translation>
+        <translation>مەن ياخشى كۆرىمەن</translation>
     </message>
     <message>
         <source>Sequential</source>
@@ -902,13 +902,13 @@
         <location filename="../UI/player/miniwidget.cpp" line="338"/>
         <location filename="../UI/player/miniwidget.cpp" line="702"/>
         <source>Loop</source>
-        <translation type="unfinished"></translation>
+        <translation>دەۋرىيلىك ئايلانما قويۇش</translation>
     </message>
     <message>
         <location filename="../UI/player/miniwidget.cpp" line="346"/>
         <location filename="../UI/player/miniwidget.cpp" line="707"/>
         <source>Random</source>
-        <translation type="unfinished"></translation>
+        <translation>ئىختىيارىي قويۇش</translation>
     </message>
 </context>
 </TS>
